Library/Homebrew/test/gpg_spec.rb

```diff
@@ -12,8 +12,15 @@
 
         shutup do
           subject.create_test_key(dir)
+          gpg = subject::GPG_EXECUTABLE
+          @version = Utils.popen_read(gpg, "--version")[/\d\.\d/, 0]
```

Comment:
Ok, I am officially out of ideas. I now know a lot more about mocks/stubs, class_doubles, partial doubles, RSpec scope, etc. Unfortunately, though it was quite an educational experience, I have yet to produce a passing test and am essentially stuck. Aid with this would be greatly appreciated.